spec/models/card_spec.rb

```diff
@@ -146,5 +146,21 @@
       end
     end
   end
+
+  context 'check levenshtein distance' do
+    before do
+      @card = user.cards.create(original_text: 'hello', translated_text: 'Привет',
```

Comment:
Line is too long. [82/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.